setup.py

```
@@ -3,24 +3,19 @@
     description="A poor man's debugger for Python.",
-    long_description=long_description,
+    long_description=open('README.md', 'r').read(),
     long_description_content_type='text/markdown',
     url='https://github.com/cool-RR/PySnooper',
     packages=setuptools.find_packages(),
-    install_requires=('decorator>=4.3.0',), 
-    tests_require=(
-        'pytest>=4.4.1',
-        'python_toolbox>=0.9.3',
-    ), 
+    install_requires=open('requirements.txt', 'r').read().split('\n'), 
```

Ruff (0.12.2)

Use a context manager for opening files

(SIM115)